providers/src/airflow/providers/standard/operators/latest_only.py

```diff
@@ -53,7 +54,7 @@ def choose_branch(self, context: Context) -> str | Iterable[str]:
         # If the DAG Run is externally triggered, then return without
         # skipping downstream tasks
         dag_run: DagRun = context["dag_run"]  # type: ignore[assignment]
-        if dag_run.external_trigger:
+        if dag_run.run_type == DagRunType.MANUAL:
```

Comment:
Ditto here about BACKFILL. Need to decide how to handle this

Reply:
so this is about short circuiting the "latest only check" when dag is externally-triggered.  I don't think it makes a lot of sense to short circuit this when backfilling since, isn't that the point of the latest only operator?